client/src/js/samples/components/__tests__/General.test.js

```diff
@@ -27,6 +27,22 @@ describe("<SampleDetailGeneral />", () => {
         const wrapper = shallow(<SampleDetailGeneral {...props} />);
         expect(wrapper).toMatchSnapshot();
     });
+
+    it("renders <MarkdownNotes /> component", () => {
+        const wrapper = shallow(<SampleDetailGeneral {...props} />);
+        expect(wrapper.find("MarkdownNotes")).toMatchSnapshot();
+    });
+
+    it("renders <EditSample /> component", () => {
+        const wrapper = shallow(<SampleDetailGeneral {...props} />);
+        expect(wrapper.find("EditSample")).toMatchSnapshot();
+    });
+
+    it("renders both Edit notes components ", () => {
+        const wrapper = shallow(<SampleDetailGeneral />);
+        expect(wrapper.find("MarkdownNotes")).toMatchSnapshot();
+        expect(wrapper.find("TextArea")).toMatchSnapshot();
+    });
```

Comment:
These are tests of the `Edit.js` module and should be at `samples/components/Edit/__tests__/Edit.test.js`.

Nothing is actually really tested here if you look at the resulting snapshots.